eva_submission/eload_utils.py

```diff
@@ -46,4 +47,27 @@ def get_file_content(file_path):
 
 def cast_list(l, type_to_cast=str):
     for e in l:
-        yield type_to_cast(e)
\ No newline at end of file
+        yield type_to_cast(e)
+
+
+def compare_sample_sets(list_of_sample_names):
+    set_of_sample_names = set(tuple(s) for s in list_of_sample_names)
+    set_of_sample_names_sorted = set([tuple(sorted(list(s))) for s in set_of_sample_names])
+    if len(set_of_sample_names) == 1:
+        return 'single set'
+    elif len(set_of_sample_names_sorted) == 1:
+        return 'unsorted single set'
+    elif are_all_elements_unique(itertools.chain(*list_of_sample_names)):
+        return 'unique sample sets'
+    elif len(set_of_sample_names_sorted) == len(list_of_sample_names):
+        return 'overlapping sample names'
```

Comment:
similar to my previous comment, i'd prefer just `else: return 'overlapping sample names'`, or if there is another possibility (i don't think there is?) an additional `else` branch.

Reply:
Same as above